<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ANANTH" failures="0" tests="1" name="lauditor.test.pages.DocumentViewFirmTest" time="0.000" errors="1" timestamp="2023-08-07T10:32:15 IST" skipped="0">
  <testcase classname="lauditor.test.pages.DocumentViewFirmTest" name="DocumentViewFirmAction_View_MergedAction_DeletePDF_PopupXClose" time="0.000">
    <error message="
Method public void lauditor.test.pages.DocumentViewFirmTest.DocumentViewFirmAction_View_MergedAction_DeletePDF_PopupXClose(java.util.HashMap) throws java.lang.InterruptedException,java.awt.AWTException,java.io.IOException requires a @DataProvider named : getData19" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Method public void lauditor.test.pages.DocumentViewFirmTest.DocumentViewFirmAction_View_MergedAction_DeletePDF_PopupXClose(java.util.HashMap) throws java.lang.InterruptedException,java.awt.AWTException,java.io.IOException requires a @DataProvider named : getData19
at org.testng.internal.Parameters.findDataProvider(Parameters.java:516)
at org.testng.internal.Parameters.handleParameters(Parameters.java:739)
at org.testng.internal.Parameters.handleParameters(Parameters.java:704)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:70)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:49)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:904)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- DocumentViewFirmAction_View_MergedAction_DeletePDF_PopupXClose -->
  <system-out/>
</testsuite> <!-- lauditor.test.pages.DocumentViewFirmTest -->
